runtime/storage_test.go

```diff
@@ -1576,11 +1576,14 @@ func TestRuntimeStorageTransfer(t *testing.T) {
 			nonEmptyKeys++
 		}
 	}
-	// 5:
+
+	// TODO: maybe retrieve and compare stored values from 2 accounts
```

Comment:
Is this still unresolved?

Reply:
Yeah, this TODO was just taken as it is from v0.42. We might want to resolve it to get more confidence before shipping, but it's not critical AFAICS. cc @fxamacker

Reply:
> Yeah, this TODO was just taken as it is from v0.42. We might want to resolve it to get more confidence before shipping, but it's not critical AFAICS. cc @fxamacker

@turbolent I agree.  We can resolve this TODO in a separate PR.